src/ploneintranet/workspace/profiles/default/workflows/ploneintranet_workspace_workflow/definition.xml

```diff
@@ -31,6 +32,12 @@
       <permission-role>Site Administrator</permission-role>
       <permission-role>TeamManager</permission-role>
     </permission-map>
+    <permission-map name="Add portal content" acquired="False">
+      <permission-role>Manager</permission-role>
+      <permission-role>Site Administrator</permission-role>
+      <permission-role>TeamManager</permission-role>
+      <permission-role>Contributor</permission-role>
+    </permission-map>
```

Comment:
We should add owner here and in several other places but that's scheduled as a separate story Assembla 430.